Makefile.js

```diff
@@ -708,6 +736,51 @@ target.checkLicenses = function() {
     });
 };
 
+target.checkGitCommit = function() {
+    var commitMsgs,
+        failed;
+
+    if (hasBranch("master")) {
```

Comment:
It differs between the build done for a [PR](https://travis-ci.org/eslint/eslint/jobs/71702836) (full repo) and done for a [push](https://travis-ci.org/eslint/eslint/jobs/71702846) (single branch only).

- PR

  ```
  git clone --depth=50 git://github.com/eslint/eslint.git eslint/eslint
  ```
- Push

  ```
  git clone --depth=50 --branch=check-commit git://github.com/eslint/eslint.git eslint/ESLint
 ```

This is why I am doing the check for master, to determine whether or not I can test for a single commit, which can only be done in a PR build.